pkg/apis/application/v1alpha1/generated.proto

```diff
@@ -496,12 +500,15 @@ message ResourceIgnoreDifferences {
   repeated string jsonPointers = 5;
 }
 
+// ResourceNetworkingInfo holds networking resource related information
 message ResourceNetworkingInfo {
   map<string, string> targetLabels = 1;
 
   repeated ResourceRef targetRef = 2;
 
   map<string, string> labels = 3;
+
+  repeated k8s.io.api.core.v1.LoadBalancerIngress ingress = 4;
```

Comment:
Minor - do we need to prefix this is `LoadBalancer`. I feel like `Ingress` does not need to be an LB.

Reply:
I've reused type from `core/v1` kubernetes package. For some reason they decided to use `LoadBalancerIngress` type name